app/controllers/candidate_interface/other_qualifications/details_controller.rb

```diff
@@ -9,6 +9,9 @@ def new
 
       unless @form.qualification_type
         @form.qualification_type = params[:qualification_type]
+        @form.other_uk_qualification_type = params[:other_uk_qualification_type] if @form.qualification_type == OtherQualificationTypeForm::OTHER_TYPE
+        @form.non_uk_qualification_type = params[:non_uk_qualification_type] if @form.qualification_type == OtherQualificationTypeForm::NON_UK_TYPE
```

Comment:
Small thing, but could these three lines sit behind a method on the form object?

Something like
`@form.set_qualification_type_fields(params[:qualification_type])`